core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -534,16 +535,16 @@ object Chunk {
         override def map[O2](f: O => O2): Chunk[O2] = indexedSeq(s.map(f))
       }
 
-  /** Creates a chunk backed by a `Seq`. */
-  def seq[O](s: Seq[O]): Chunk[O] = s match {
+  /** Creates a chunk backed by a mutable `Seq`. */
+  def seq[O](s: GSeq[O]): Chunk[O] = s match {
     case a: collection.mutable.WrappedArray[O] =>
       array(a.array.asInstanceOf[Array[O]])
     case v: Vector[O]      => vector(v)
     case ix: IndexedSeq[O] => indexedSeq(ix)
     case _ =>
       if (s.isEmpty) empty
       else if (s.tail.isEmpty) singleton(s.head)
-      else buffer(collection.mutable.Buffer(s: _*))
+      else buffer(collection.mutable.Buffer((s.toSeq): _*))
```

Comment:
Would a match for `Buffer` to go through `buffer` without `s.toSeq` be worthwhile?  I doubt this gets passed `Buffer`s often, but it's what jawn-fs2 always sends.  I may end up special casing it there, even after this fix.

Reply:
Probably a good idea.

The `s.toSeq` here is weird -- I think that will no longer be necessary as of 2.13.0-RC1. I'll add a comment to double check that.